src/mpid/ch4/subconfigure.m4

```diff
@@ -373,7 +373,7 @@ esac
 # - enable-thread-cs=per-vci
 #
 if test "$enable_ch4_mt" != "direct"; then
-    if test "${with_zm_prefix}" == "no" -o "${with_zm_prefix}" == "none" -o "${izem_queue}" != "yes" ; then
+    if test "${with_zm_prefix}" == "no" -o "${with_zm_prefix}" == "none" -o "${enable_izem_queue}" != "yes" ; then
         AC_MSG_ERROR([Multi-threading model `${enable_ch4_mt}` requires izem queue. Set `--enable-izem={queue|all} --with-zm-prefix` and retry.])
```

Comment:
Need update the error message as well

Reply:
@sagarth I believe @jain-surabhi-23 have an internal commit. Please feel free to replace my commits with your internal commit.